tests/setup_local_db/ddl_results_and_cdm.sql

```diff
@@ -101,44 +101,6 @@ FROM omop.observation ob
 INNER JOIN omop.concept concept ON concept.CONCEPT_ID=ob.OBSERVATION_CONCEPT_ID
 WHERE concept.CONCEPT_CLASS_ID='MVP Continuous' or concept.CONCEPT_ID=2000007027;
 
-CREATE VIEW omop.DATA_DICTIONARY AS
```

Comment:
this view still seems to be used in a few places

Reply:
We replaced this view with a table of the same name for performance reasons.

Reply:
so this line is using the table now:
https://github.com/uc-cdis/cohort-middleware/blob/d7010b774d51036d6fc06dec14e005e9b639519e/models/datadictionary.go#L147-L148 
?